api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -911,4 +911,39 @@ public PersonAddress build() {
 			return personAddress;
 		}
 	}
+
+	/**
+	 * @see Person#Person(Person)
+	 * @verifies deep copy all collections
+	 */
+	@Test
+	public void Person_shouldDeepCopyAllCollections() throws Exception {
```

Comment:
I know :) but it was created like that by unit test plug-in. It's because it is testing copying consturctor, which starts with upper case. Maybe it should stay like that to match tested function name?